doc/src/site/sphinx/tutorials/azure_hdi.rst

```diff
@@ -0,0 +1,78 @@
+Using Sparkling Water with Microsoft Azure HDInsight - Beta
+------------------------------------------------------------
+
+Microsoft Azure provides an important collection of cloud services, such as serverless computing, virtual machines, storage options, networking, and much more. Azure provides the tools for a user to create a Data Science environment with Sparkling Water.
+
+This section describes the H2O Application for HDInsight on Microsoft Azure.
+
+**Note**: This feature is currently in Beta and should be used for testing purposes only.
+
+H2O Artificial Intelligence for Azure HDInsight
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The H2O Artificial Intelligence for Azure HDInsight is an application you can install during the creation of a new HDInsight cluster on Azure. This solution will install Sparkling Water on your Spark cluster, allowing you to exploit all the benefits from both Spark and H2O. The cluster can access data from Azure Blob storage and/or Azure Data Lake Store in addition to all the standard data sources that H2O supports. It also provides Jupyter Notebooks with pre-baked H2O examples for an easy jumpstart.
+
+Create the H2O AI for Azure HDInsight
+'''''''''''''''''''''''''''''''''''''
+
+**Requirement**: Python 2.7. Sparkling Water currently works only on Python 2.7; it will not work on PySpark3 kernel.  
+
+Follow the steps below to create a new H2O Artificial Intelligence for Azure HDInsight.
+
+1. In your Azure portal at `https://portal.azure.com <https://portal.azure.com>`__, search for H2O, and select **H2O Artificial Intelligence for HDInsight**.
+
+2. Click the **Create** button, and follow the UI instructions.
+
+   **Note**: H2O for HDInsight is exclusively for Spark HDI clusters version 3.5 (HDI v3.5).
+
+   .. figure:: ../images/azure_select_h2o_hdinsight.png
+      :alt: Select H2O Artificial Intelligence for HDInsight
+
+3. In the next screen, under **Basics**, change the Cluster Type to Spark 2.0.2. Sparkling Water is currently configured to work only on Spark 2.0 and above.
```

Comment:
I know that it's not related strictly to documentation change, but we should reference only 2.0 from Documentation for Spark 2.0

The ideal solution for me -> prepare HDInsight also for Spark 2.3, 2.2 and 2.1

Reply:
Can I use a version tag for this?

Reply:
Yup, that would be great - version substitution for both Spark and Sparkling Water. Let me just check that the HDInsight image is available for these versions as well

Reply:
So if you can just use the substitution for Spark and Sparkling Water version that would be great, thanks!